tests/pipelines/test_base_pipeline.py

```diff
@@ -43,6 +43,51 @@ def some_pipeline(step_):
     return pipeline_instance
 
 
+def test_initialize_pipeline():
+    @step
+    def empty_step():
+        pass
+
+    @pipeline
+    def some_pipeline(step_1, step_2):
+        pass
+
+    # initialize with args
+    some_pipeline(empty_step(), empty_step())
```

Comment:
Perhaps we can add these as [fixtures](https://www.tutorialspoint.com/pytest/pytest_fixtures.htm) instead? Would be nice to give them descriptive names and also make them reusable across different scenarios.